rasterio/plot.py

```diff
@@ -228,7 +228,7 @@ def reshape_as_raster(arr):
     return im
 
 
-def show_hist(source, bins=10, masked=True, title='Histogram', ax=None, label=None, **kwargs):
+def show_hist(source, bins=10, masked=True, title='Histogram', ax=None, label=None, hist_range=None, **kwargs):
```

Comment:
@lpinner I'd prefer to rename `hist_range` to `range` if we can, for harmony with the matplotlib API.